src/include/nxt/EnumClassBitmasks.h

```diff
@@ -53,16 +53,23 @@ namespace nxt {
     struct LowerBitmask<T, typename std::enable_if<IsNXTBitmask<T>::enable>::type> {
         static constexpr bool enable = true;
         using type = T;
-        static T Lower(T t) {return t;}
+        constexpr static T Lower(T t) {
```

Comment:
If possible, could you land this commit separately, since it's just constexpr and whitespace changes? I see it's in the middle, but you could probably reorder and squash the other two.

Reply:
Done